### DISCUSSION TITLE
The original power source is broken, can I substitute it?

### DISCUSSION THREAD

**Discussion by Capynetics** [HumanSupportRobot/QandA#108] [Q&A]
The original power source that came with the robot is not working anymore.
This is the original model: https://www.mouser.com/datasheet/2/141/fhg-sx-u-30236.pdf
I am thinking about substituting it with the MS-600W-48 (https://i.imgur.com/EJMHamj.png)
They both have similar ratings for current, voltage, and power, so I am confident that the substitution can be made but I would like to hear other opinions before taking any actions.

**Comment by yoshimi-iyoda** [comment]
Since it has a variable output from 0 to 48V, I think it will be fine if you use it at a 30V output.

**Comment by MaDiBmDaD** [comment]
Just curious—what led you to identify it as the PSU for the HSR? looks a bit different...

**Reply by Capynetics** [reply]
The FHG SX-U1 looks exactly like the Power Source Unity use by my HSR. The difference is that the frontal part, where the connectors are, had a custom made black plastic cover. But once I removed the cover they were identical. Maybe yours have this cover too?
